server/models/Profile.js

```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose'
+
+const profileSchema = mongoose.Schema({
+	profile_photo: {
+    type: String,
+  },
+	first_name: {
+    type: String,
+    required: true,
+  },
+	last_name: {
+    type: String,
+    required: true,
+  },
+	gender: {
+    type: String,
+    enum: ['male', 'female'],
```

Comment:
```suggestion
    enum: ['male', 'female', 'other'],
```
Why don't we consider adding 'other' for individuals who identify as non-binary